apps/condo/domains/property/components/panels/Builder/forms/UnitForm.tsx

```diff
@@ -0,0 +1,250 @@
+/** @jsx jsx */
+import React, { useState, useRef, useEffect, useCallback, useMemo } from 'react'
+import { useIntl } from '@condo/next/intl'
+import { BuildingUnitSubType } from '@app/condo/schema'
+import { Row, Col, Space, Typography } from 'antd'
+import Select from '@condo/domains/common/components/antd/Select'
+import Input from '@condo/domains/common/components/antd/Input'
+import Checkbox from '@condo/domains/common/components/antd/Checkbox'
+import { Button } from '@condo/domains/common/components/Button'
+import { DeleteFilled } from '@ant-design/icons'
+import { jsx } from '@emotion/react'
+import {
+    IPropertyMapModalForm,
+    MODAL_FORM_ROW_GUTTER,
+    MODAL_FORM_ROW_BUTTONS_GUTTER,
+    MODAL_FORM_CHECKBOX_STYLE,
+    INPUT_STYLE,
+    ERROR_TEXT_STYLE,
+    BUTTON_SPACE_SIZE,
+    FormModalCss,
+} from './BaseUnitForm'
+import { MapEditMode } from '@condo/domains/property/components/panels/Builder/MapConstructor'
+
+const { Option } = Select
+
+const UnitForm: React.FC<IPropertyMapModalForm> = ({ builder, refresh }) => {
+    const intl = useIntl()
+    const mode = builder.editMode
+    const SaveLabel = intl.formatMessage({ id: mode === MapEditMode.EditUnit ? 'Save' : 'Add' })
```

Comment:
perhaps it would be better to use `SaveLabel` and `AddLabel` separately and by condition to use this or that label?